engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/NKGlobalParameters.scala

```diff
@@ -0,0 +1,34 @@
+package pl.touk.nussknacker.engine.flink.api
```

Comment:
It looks like all this flags are not Flink specific but only paring them from `ExecutionConfig`? Maybe it should be in common api for purpose of other engines?

Reply:
... but then will be nice to have also `NkFlinkGlobalParameters(common: NkGlobalParameters, ...)` to be able to add some Flink's specific :)  If you prefer to just suffix it with `NkFlink...` it will be also fine.

Reply:
I'll stick with Flink specific for now. I don't see much usage outside, and we would have to circumvent somehow subclassing GlobalJobParameters...